packages/gasket-plugin-happyfeet/package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "gasket-plugin-happyfeet",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "directories": {
+    "lib": "lib"
+  },
+  "scripts": {
+    "test": "cross-env NODE_OPTIONS='--unhandled-rejections=strict' jest"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "MIT",
```

Add the [bugs](https://docs.npmjs.com/cli/v6/configuring-npm/package-json?v=true#bugs) property:

```suggestion
  "license": "MIT",
    "bugs": {
    "url": "https://github.com/godaddy/gasket/issues"
  },
```